Formula/diffoscope.rb

```diff
@@ -14,7 +15,6 @@ class Diffoscope < Formula
     sha256 cellar: :any_skip_relocation, mojave:        "93271ba0f90255a8a93e2bd4d75701b9e4404fed70c27bb4dd97bcebe6d386a2"
   end
 
-  depends_on "gnu-tar"
```

Comment:
Kinda weird that the formula depends on both. Maybe doing so actually adds functionality? If it does maybe we can put this `depends_on` in an `on_macos` block.

Reply:
Yes, I am unsure why it depended on both. Maybe it's a typo, or a small oversight. I want to see if it builds, and ship it as-is without `gnu-tar`. If anyone complains that something is broken we can re-evaluate the situation.

Reply:
There are some differences: https://unix.stackexchange.com/q/101561/304692

Given what diffoscope does, I think it might be better to keep both dependencies.

Reply:
This does mean that `diffscope` can not be installe on Linux in that case, because it requires 2 conflicting libraries.

Reply:
Not even if `depends_on "gnu-tar"` is in an `on_macos do` block?

Reply:
Right, I can do that. Better than nothing.

Reply:
On the other hand, the fact that the two libraries are conflicting might mean that `diffoscope` is actually really only using one of them 😅 So maybe I take back my comment about putting it a `on_macos do` block and we can just wait to see if someone complains.

Reply:
But I see you've pushed the changes already, so it's fine, no need to change it anymore.